src/sdam/description/topology/mod.rs

```diff
@@ -286,6 +298,81 @@ impl TopologyDescription {
         }
     }
 
+    /// Updates the topology's logical session timeout value based on the server's value for it.
+    fn update_session_support_status(&mut self, server_description: &ServerDescription) {
+        if !server_description.server_type.is_data_bearing() {
+            return;
+        }
+
+        if server_description.server_type == ServerType::Standalone {
+            self.session_support_status = SessionSupportStatus::Unsupported {
+                logical_session_timeout: server_description
```

Comment:
Kind of strange that a standalones still report this field given that they don't support sessions

Reply:
Yeah, it's pretty annoying. I filed [DRIVERS-1230](https://jira.mongodb.org/browse/DRIVERS-1230) to talk about allowing drivers to ignore this value.